### DISCUSSION TITLE
Growatt forcing to set their own IP to the ShineStick since tonight 24-03-28

### DISCUSSION THREAD

**Discussion by r4tt4** [johanmeijer/grott#541] [General]
Hi *,

This morning I was some kind of wondering why the Growatt does not have any Voltage on PV in my monitoring. So system was working normal and also the ShinePhone or web based server.growatt.com show everything running fine. 
Just found out after looking at my second Growatt, where I blocked all commands cause not necessary, that Growatt sends a command since this morning and got blocked: "Grott: Record blocked:  0118". 
So my 1st Growatt - not blocked - was forced since received the command 0118 to send metrics to Growatt directly and no longer through grott. I was able to set the local grott IP back in the WebUI, but that works only for some minutes and again the command 0118 set the offical IP. Now grott is configured to block all commands and everything is working again. Monitoring an Metrics are produced.

Anybody else seen this behavior?

And I got a question: I have two parameters in the config file:
blockcmd = True 
#noipf = True

Is it possible to block ONLY changing the IP from outside like that:
blockcmd = False
noipf = False
So allowing change of parameters except of changing the IP address. That would re-enable configure stuff from the ShinePhone App without getting cut off to my internal network again.

Regards
Björn

**Comment by KasperEdw** [comment]
Same here, quite annoying. 
I changed the IP back to the grott server and set blockcmd = True
It has been running fine since I did it a couple of hours ago. 

Best wishes, 
Kasper

**Comment by johanmeijer** [comment]
See also: #542
